tests/test_geometry.py

```diff
@@ -103,3 +103,27 @@ def test_antimeridian(self):
         self.assertTrue(
             all(lon >= 0 for lon in item_lons)
             or all(lon <= 0 for lon in item_lons))
+
+    def test_presplit_antimeridian_normalize(self):
+        """Test that an item with geometry already split along the antimeridian
+        does not trigger the stactools antimeridian MultiPolygon value error.
+        Use the NORMALIZE strategy.
+        """
+        mtl_xml_href = TEST_GEOMETRY_PATHS["presplit_antimeridian"]
+        item = create_stac_item(mtl_xml_href,
+                                use_usgs_geometry=True,
+                                legacy_l8=False,
+                                antimeridian_strategy=Strategy.NORMALIZE)
+        self.assertEqual(item.geometry["type"], "Polygon")
```

Comment:
If possible, it'd be good to get a unit test against the expected normalized polygon.

Reply:
I can do that.

Reply:
done